src/components/styles/Layout.tsx

```diff
@@ -64,8 +64,10 @@ export const TableControlOuterRow = styled(TableControlInnerRow)`
 
 export const PageButtonRow = styled.div`
   display: flex;
-  justify-content: space-between;
-  width: 350px;
+  button {
+    margin-left: 24px;
+  }
+  flex-wrap: wrap;
```

Comment:
We use this component for task and patch page. Since the number of buttons have diverged, applying a margin-left keeps spacing between the buttons consistent between the pages. I also allow the buttons to stack to make better use of screen real estate. 
![Screen Shot 2021-11-02 at 12 54 19 PM](https://user-images.githubusercontent.com/10734386/139910272-9f472b16-8f91-4eaa-b44b-cc7f5ec32b79.png)

Reply:
The buttons will take the height of the container for the copy on the left. I changed the `width` style of the copy container to `max-width` so the buttons don't stack until they really have to